packages/hoppscotch-data/src/environment/index.ts

```diff
@@ -165,7 +165,7 @@ export const translateToNewEnvironment = (x: any): Environment => {
   if (x.v && x.v === EnvironmentSchemaVersion) return x
 
   // Legacy
-  const id = x.id ?? ""
+  const id = x.id ?? uniqueID()
```

Comment:
We could maybe replace this with `||` to ensure IDs as empty strings fall back to the unique ID variant.